.pre-commit-config.yaml

```diff
@@ -195,6 +195,15 @@ repos:
           - "4"
         files: ^chart/values\.schema\.json$|^chart/values_schema\.schema\.json$
         pass_filenames: true
+      - id: pretty-format-json
+        name: Serialization schema
+        args:
+          - --autofix
+          - --no-sort-keys
+          - --indent
+          - "2"
```

Comment:
yeah i'll split it out.

but yeah it was previously not processed in pre-commit.  but it should be to ensure consistent formatting, even if that means slightly less pretty.
and yeah i just chose indent of 2 because that's what the file already had.

Reply:
ok updated